doc/TILESET.md

```diff
@@ -1,141 +1,191 @@
-
 ### `tile_info.json`
-Each compositing tileset *must* have a `tile_info.json`, laid out like so:
 ```
 [
   {
     "width": 32,
-    "pixelscale": 1,
-    "height": 32
-  },
-  {
-    "1_tiles_32x32_0-5199.png": {}
-  },
-  {
-    "2_expan_32x32_5200-5391.png": {}
-  },
-  {
-    "3_tree_64x80_5392-5471.png": {
-      "sprite_offset_x": -16,
-      "sprite_offset_y": -48,
+    "height": 32,
+    "pixelscale": 1
+  }, {
+    "tiles.png": {}
+  }, {
+    "expan.png": {}
+  }, {
+    "tree.png": {
+      "sprite_width": 64,
       "sprite_height": 80,
-      "sprite_width": 64
+      "sprite_offset_x": -16,
+      "sprite_offset_y": -48
+    }
+  }, {
+    "fillerhoder.png": { 
+      "source": "https://github.com/CleverRaven/Cataclysm-DDA/tree/b2d1f9f6cf6fae9c5076d29f9779e0ca6c03c222/gfx/HoderTileset",
+      "filler": true 
+    }
+  }, {
+    "fallback.png": {
+      "fallback": true
     }
-  },
-  {
-    "4_fallback_5472-9567.png": { "fallback": true }
   }
 ]
 ```
-The first dictionary is mandatory, and gives the default sprite width and sprite height for all tilesheets in the tileset.  Each of the image directories must have a separate dictionary, containing the tilesheet png name as its key.  If the tilesheet has the default sprite dimensions and no special offsets, it can have an empty dictionary as the value for the tilesheet name key.  Otherwise, it should have a dictionary of the sprite offsets, height, and width.
 
-A special key is `"fallback"` which should be `true` if present.  If a tilesheet is designated as fallback, it will be treated as a tilesheet of fallback ASCII characters.  `compose.py` will also compose the fallback tilesheet to the end of the tileset, and will add a "fallback.png" to `tile_config.json` if there is no `"fallback"` entry in `tile_info.json`.
+The first object defines the default sprite size.
 
-A special is `"filler"` which should be `true` if present.  If a tilesheet is designated as filler, entries from its directory will be ignored if an entry from a non-filler directory has already defined the same id.  Entries will also be ignored if the id was already defined by in the filler directory.  Also, pngs from a filler directory will be ignored if they share a name with a png  from a non-filler directory.  A filler tilesheet is useful when upgrading the art in a tileset: old, low-quality art can be placed on filler tilesheet and will be automatically replaced as better images are added to the non-filler tilesheets.
+Each tilesheet directory must have a corresponding object with single key being the tilesheet output filename.  Empty object as value here means that the tilesheet has default sprite size and no offsets. It can have overriding values under `sprite_width`, `sprite_height`, `sprite_offset_x`, `sprite_offset_y` keys.
```

Comment:
```suggestion
Each tilesheet directory must have a corresponding object with a single key, which must be the tilesheet output filename.  Passing an empty object as the value here means that the tilesheet uses the default sprite size and has no offsets. It can have overriding values under `sprite_width`, `sprite_height`, `sprite_offset_x`, `sprite_offset_y` keys.
```

Reply:
Preparing to batch-commit your suggestions, thank you! The filename is _defined_ by the key here. Are you sure "must be" fits?

Reply:
Ah, the original text didn't make clear which way was conducted. Change that for "which will become the tilesheet output filename."